snap7/client.py

```diff
@@ -31,25 +32,25 @@ class Client:
     A snap7 client
     """
 
-    def __init__(self):
+    def __init__(self) -> None:
```

Comment:
I think `None` is the default return type, so you can save yourself the time to add it :)